osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -100,6 +100,7 @@ protected override void InitialiseDefaults()
 
             Set(OsuSetting.IncreaseFirstObjectVisibility, true);
             Set(OsuSetting.GameplayDisableWinKey, true);
+            Set(OsuSetting.GameplayDisableOverlayActivation, true);
```

Comment:
seeing that this is given a default of `true` this tip might need wording adjustment

https://github.com/ppy/osu/blob/fd56db21568a7c71e4390a4b20a69e0922538a88/osu.Game/Screens/Menu/Disclaimer.cs#L193

Reply:
IMO, I don't think the tip needs to be adjusted. I mean, the tip is still half valid in the sense that you can change the skin while playing but you'll have to pause to open the settings and switch skins, but I guess that's what most of players (to not say all) would do.

Reply:
mostly I'd just mention that pausing is necessary to avoid people claiming that the tip is incorrect:

> All settings are dynamic and take effect in real-time. Try pausing and changing the skin while playing!